css/components/_btn.scss

```diff
@@ -0,0 +1,77 @@
+
+.btn {
+  border: 1px solid transparent;
+  border-radius: .25rem;
+  color: $white;
+  cursor: pointer;
+  display: inline-block;
+  font-weight: 400;
+  padding: 1px;
+  position: relative;
+  text-align: center;
+  user-select: none;
+  vertical-align: middle;
+
+  a {
```

Comment:
What's the use case for an `a` element inside an element having `btn` as a class?